shared/constants/metametrics.js

```diff
@@ -175,8 +175,6 @@
  * identify the new number_of_tokens trait
  * @property {'opensea_api_enabled'} OPENSEA_API_ENABLED - when the OpenSea API is enabled
  * we identify the opensea_api_enabled trait
- * @property {'three_box_enabled'} THREE_BOX_ENABLED - when 3box feature is
```

Comment:
if my suggestion to turn the trait t ofalse instead of removing it is accepted, then keep these constants/types but notate they are deprecated.

Reply:
Done: 3c3475bf20f1a2ac4445b30bc743632ca2969f23